core/Symbols.cc

```diff
@@ -155,6 +155,13 @@ SymbolRef Symbol::ref(const GlobalState &gs) const {
     return SymbolRef(gs, type, distance);
 }
 
+bool SymbolRef::isField(const GlobalState &gs) const {
+    return isFieldOrStaticField() && data(gs)->isField();
+}
+bool SymbolRef::isStaticField(const GlobalState &gs) const {
+    return isFieldOrStaticField() && data(gs)->isStaticField();
+}
```

Comment:
Why are these two special?

Reply:
Good question. I think it would work in the header too. I can change that.

Reply:
Ah, it's in the cc file because we don't pull in `Symbols.h` so I can't access the incomplete symbol type.